whylog/client/const.py

```diff
@@ -0,0 +1,2 @@
+class bufsize_consts:
+    STANDARD_BUF_SIZE = 8192
```

Comment:
It would seem that currently this value is not set appropriately. Correct it or create a new issue to do it later, if it is impossible now.